### PR TITLE
use frozendict for migrator_uid.

### PR DESCRIPTION
**Pull request by CJ-Wright**
This simplifies a bunch of really annoying syntax and allows us to get
the correct PRed_json every time

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Need to move from update to `.copy`

**Comment by scopatz** [comment]
Code looks fine but it is failing CI for some reason

**Comment by CJ-Wright** [comment]
I need to use copy rather than update because frozen dict Arya immutable. Skidoo I need to coordinate this PR with changes to the graph so I'll halt ops while that happens.

File: conda_forge_tick/migrators.xsh [hunk 1]
```diff
@@ -282,7 +286,7 @@ class Version(Migrator):
 
     def migrator_uid(self, attrs):
         n = super().migrator_uid(attrs)
-        n.update({'version': attrs["new_version"]})
+        n = n.copy(**{'version': attrs["new_version"]})
```
Comment (scopatz) [inline conda_forge_tick/migrators.xsh]: Wouldn't it be cleaner to do `n.copy(version=attrs["new_version"])`

File: conda_forge_tick/migrators.xsh [hunk 2]
```diff
@@ -612,7 +616,7 @@ class Rebuild(Migrator):
 
     def migrator_uid(self, attrs):
         n = super().migrator_uid(attrs)
-        n.update({'name': self.name})
+        n = n.copy(**{'name': self.name})
```
Comment (scopatz) [inline conda_forge_tick/migrators.xsh]: same here, isn't this just `n.copy(name=self.name)`

**Comment by CJ-Wright** [comment]
ping @scopatz

**Comment by CJ-Wright** [comment]
Thank you

**Pull request merged**